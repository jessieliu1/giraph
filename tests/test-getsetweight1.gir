int main() {
	wegraph<int> g = [A:1 -{5}- B -{7}- C];
	g.add_edge(C, A, 9);

	for_edge(e : g) {
		int w = e.weight();
		g.set_edge_weight(e.from(), e.to(), w+5);
	}

	for_edge(e : g) {
		print(e.weight());
		print(g.get_edge_weight(e.from(), e.to()));
		print(g.get_edge_weight(e.to(), e.from()));
	}
	return 0;
}