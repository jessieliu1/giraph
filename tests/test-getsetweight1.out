10
10
10
14
14
14
12
12
12
